l10n_it_fatturapa_in/models/attachment.py

```diff
@@ -4,6 +4,32 @@
 SELF_INVOICE_TYPES = ("TD16", "TD17", "TD18", "TD19", "TD20", "TD21", "TD27", "TD28")
 
 
+class Attachment(models.Model):
+    _inherit = "ir.attachment"
+
+    @api.model
+    def check(self, mode, values=None):
+        for attachment in self:
```

Comment:
Giusto una nota per dire che normalmente nei metodi `api.model` `self` è un recordset vuoto, questo metodo però viene anche chiamato su dei record, ad esempio in https://github.com/odoo/odoo/blob/d752abb24d9b25bb6d10e411961447c8c8e28bba/odoo/addons/base/models/ir_attachment.py#L503